cookbooks/aws-parallelcluster-environment/test/controls/mount_shared_spec.rb

```diff
@@ -9,10 +9,10 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'tag:config_mount_shared_configured' do
+control 'mount_shared_configured' do
```

Comment:
[Nitpick] I would call this control "mount_shared_mounted" or "headnode_shared_mounted" instead of "mount_shared_configured" because the control is not testing that the mounts are configured, but that they are actually mounted.

Reply:
Sure, I didn't modify the tests I was just moving them from root folder to functional specific cookbooks. Fixed.